vis_server/protocols/visualizers/vcs_vis.py

```diff
@@ -20,17 +20,34 @@ def __init__(self, *arg, **kw):
         self._plot.graphics_method = vcs.getisofill() # default
         self._plot.template = vcs.elements['template']['default'] # default
 
-    def render(self, opts={}):
-        super(VcsPlot, self).render(opts)
+    def toJSON(self, imageData, state, mtime, size, format, globalId, localTime, workTime):
+        reply = {}
+        reply['image'] = imageData
+        reply['state'] = state
+        reply['mtime'] = mtime
+        reply['size'] = size
+        reply['format'] = format
+        reply['global_id'] = globalId
+        reply['localTime'] = localTime
+        reply['workTime'] = workTime
+        return reply
+
+    def render(self, cfg):
+        self.getView().SetSize(self._width, self._height)
```

Comment:
It doesn't look to me like resizing ever happens in this PR.  `self._width` and `self_height` are [fixed](https://github.com/UV-CDAT/cdatweb/pull/84/files#diff-9cedbbb0cdad97561c5081f8c4d1e39eR16) at `900 x 600` because you aren't calling the super method.  Also, the sizes passed through to vcs and to javascript are hard coded to  `500 x 500`.